packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -212,47 +212,6 @@ void main() {
         expect(getListHeight(), kDraggingListHeight);
       });
 
-      testWidgets('Vertical drop area golden', (WidgetTester tester) async {
```

Comment:
What did these tests verify? Why do we no longer need to guarantee that this use case works?

Reply:
It is not entirely clear from the code, but from the title 'Vertical drop area golden', I presume it was trying to test that there was a drop area when an item was dragged. However, looking at this code it creates a three item list, starts dragging the middle item (blue) and then takes a screen shot. However, as the item has not yet moved, it is sitting over the drop area, so the golden image is just a list with three items in it. Not sure what it was trying to verify here.  Removing the overlay would mean the item is no longer in the list view's repaint boundary, so the golden would have to be updated, but again, I am not sure what value it provides.

Reply:
> However, as the item has not yet moved, it is sitting over the drop area, so the golden image is just a list with three items in it.

Is that maybe what the test was testing? If you pick up an item it starts out to render at the correct location (i.e. at its original location in the list).

Reply:
Yeah, as I was explaining this above, that thought occurred to me as well. I will update the test to show the missing item during the drag and update the golden for it. Thx for the input.

Reply:
I just pushed a commit that re-added these tests, but updated them for capturing a drag in progress.